test/cli/import-jdl.spec.js

```diff
@@ -383,26 +383,21 @@ describe('JHipster generator import jdl', () => {
         beforeEach(done => {
             testInTempDir(dir => {
                 fse.copySync(path.join(__dirname, '../templates/import-jdl'), dir);
-                importJdl(['deployments.jdl'], { skipInstall: true }, env, mockFork(done, 4));
+                importJdl(['deployments.jdl'], { skipInstall: true }, env, mockFork(done, 3));
             });
         });
 
         it('creates the deployments', () => {
             assert.file([
                 path.join('docker-compose', '.yo-rc.json'),
                 path.join('kubernetes', '.yo-rc.json'),
-                path.join('openshift', '.yo-rc.json'),
-                path.join('rancher-compose', '.yo-rc.json')
+                path.join('openshift', '.yo-rc.json')
             ]);
         });
         it('calls deployment generator', () => {
-            expect(subGenCallParams.count).to.equal(4);
-            expect(subGenCallParams.commands).to.eql([
-                'jhipster:docker-compose',
-                'jhipster:kubernetes',
-                'jhipster:openshift',
-                'jhipster:rancher-compose'
-            ]);
+            const invokedSubgensCount = 3;
+            expect(subGenCallParams.count).to.equal(invokedSubgensCount);
+            expect(subGenCallParams.commands).to.eql(['jhipster:docker-compose', 'jhipster:kubernetes', 'jhipster:openshift']);
```

Comment:
@PierreBesson Maybe you should calculate the count based on this list, so next time if we add/remove 1 subgen it is just needed to add/remove from the list like you do in this PR?

Reply:
Good idea, I just did that !